### PR TITLE
[REG-1223] Make git the preferred approach for importing the SDK

### PR DESCRIPTION
**Pull request by vontell**
Right now, we instruct users to download a package directly, and then only mention git as a secondary option. From the CI/CD work, I found that the direct download option is problematic (i.e. the RG SDK is then not found), and so this PR makes the Git option the preferred approach. Also adds a section to the home page of the SDK for getting started quickly.

### CODE REVIEW

File: docs/studios/unity/tutorials/first_tutorial.md [hunk 1]
```diff
@@ -61,21 +61,19 @@ the game loop, one for controlling the character, and one for controlling the po
...
-and import the package by clicking **Add package from disk** in the **Package Manager** window (this window can be opened
-in the **Window > Package Manager** menu ). 
-- Note that this package utilizes TextMeshPro.  If you are prompted by Unity to add TextMeshPro assets to your project, please add them.
-- Note that this package installs a dependency of Newtonsoft Json (https://docs.unity3d.com/Packages/com.unity.nuget.newtonsoft-json@3.1/manual/index.html) for serializing/deserializing Json message payloads.
-
-![Screenshot of the package imported into the project](first_bot_tutorial_images/tutorial_1_package.png)
-
-You can also download the project directly by clicking the plug button and the **Add package from git URL** option, and
-pasting in the following URL.
+Our package is available via GitHub [here](https://github.com/Regression-Games/RGUnityBots). Clone and import the repository by
```
Comment (abeizer) [inline docs/studios/unity/tutorials/first_tutorial.md:64]: Do they still need to clone it?

File: docs/studios/unity/tutorials/first_tutorial.md [hunk 2]
```diff
@@ -61,21 +61,19 @@ the game loop, one for controlling the character, and one for controlling the po
...
-pasting in the following URL.
+Our package is available via GitHub [here](https://github.com/Regression-Games/RGUnityBots). Clone and import the repository by 
+clicking **Add package from git URL** in the **Package Manager** window (this window can be opened
+in the **Window > Package Manager** menu) and pasting in the following URL.
 
 ```
 https://github.com/Regression-Games/RGUnityBots.git
 ```
 
+- Note that this package utilizes TextMeshPro.  If you are prompted by Unity to add TextMeshPro assets to your project, please add them.
```
Comment (RG-nAmKcAz) [inline docs/studios/unity/tutorials/first_tutorial.md:72]: @rcornwall-reg Be sure to have a PR in Regression Docs to update this line as part of your other work.

File: docs/studios/unity/tutorials/first_tutorial.md (hunk 1 above)
Comment (RG-nAmKcAz) [reply docs/studios/unity/tutorials/first_tutorial.md:64]: Will approve pending resolution of this update

Comment (vontell) [reply docs/studios/unity/tutorials/first_tutorial.md:64]: Good call, changing to just "Import"

**Review by abeizer — APPROVED** [review]

**Review by RG-nAmKcAz — APPROVED** [review]

**Pull request merged**